src/Identity/UI/src/build/Microsoft.AspNetCore.Identity.UI.targets

```diff
@@ -0,0 +1,23 @@
+<Project>
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.AspNetCore.StaticWebAssets.targets"/>
+
+  <Target Name="ValidateIdentityUIFrameworkVersion" BeforeTargets="Build">
+
+    <PropertyGroup>
+      <_IdentityUIUnknownVersionErrorMessage>The 'IdentityUIFrameworkVersion' '$(IdentityUIFrameworkVersion)' in '$(MSBuildProjectFile)' is not valid. Valid versions are 'Bootstrap4'.
+      </_IdentityUIUnknownVersionErrorMessage>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <_IdentityUIActiveWarnings Include="IdentityUI002" Exclude="$(NoWarn)" />
+    </ItemGroup>
+
+    <Error
+           Code="IdentityUI002"
+           Text="$(_IdentityUIUnknownVersionErrorMessage)"
+           File="@(MSBuildThisProjectFile)"
+           Condition="'$(IdentityUIFrameworkVersion)' != 'Bootstrap4' and '%(_IdentityUIActiveWarnings.Identity)' == 'IdentityUI002'" />
```

Comment:
or maybe not...

Reply:
This should use the same property we used in the .NET 5.0 package